rust/private/rust.bzl

```
@@ -703,6 +703,13 @@ _rust_test_attrs = {
             ["Make variable"](https://docs.bazel.build/versions/master/be/make-variables.html) substitution.
         """),
     ),
+    "harness": attr.bool(
+        mandatory = False,
+        default = True,
+        doc = _tidy("""\
+            Whether to use libtest (defaults to True).
```

```suggestion
            Whether to use libtest.
```

Stardoc will render the default, we don't need to also write it into the docs.